PlayCover.xcodeproj/project.pbxproj

```diff
@@ -500,6 +501,8 @@
 				"fa-IR",
 				"zh-Hant",
 				ar,
+				"pt-br",
```

Comment:
Is it right to have this "duplicated"? `pt-br` and `pt-BR`.

Reply:
This was automatically added when I created Localizable.strings inside the pt-br.lproj folder, but I've seen it used both ways here.